### PR TITLE
Adds v.max validator function

### PR DESCRIPTION
**Pull request by cdeborja**
Delivers:
- `v.max` validator, which validates if a passed in value doesn't exceeded the max value
- added tests for function

### CODE REVIEW

File: test/lib/check.spec.js [hunk 1]
```diff
@@ -704,6 +704,49 @@ describe('awesomize/lib/check', () => {
     });
 
   });
+
+  describe('::max', () => {
+
+    it(`should return Check.MSG.MAXIMUM_EXCEEDED if value is not equal
```
Comment (Bellk17) [inline test/lib/check.spec.js]: I don't think the logic is quite right in this statement...

**Review by Bellk17 — APPROVED** [review]
:shipit:

**Review by travistrue2008 — APPROVED** [review]
:shipit:

**Review by SeanTobin — APPROVED** [review]
:shipit:

**Pull request merged**